src/client/components/SelectionPage/Selectionpage.css

```diff
@@ -0,0 +1,46 @@
+.seletionPage {
+  width: 100%;
+  text-align: center;
+}
+.logo {
```

Comment:
All other selectors in this file should start with `.selection-page` to not clash with other styles on other pages.